orttraining/orttraining/test/python/orttraining_test_ortmodule_api.py

```diff
@@ -28,6 +28,7 @@
                                             LogLevel,
                                             _fallback,
                                             _graph_execution_manager)
+import onnxruntime.training.ortmodule as training_ortmodule
```

Comment:
`training_ortmodule` gives the impression that there are multiple ortmodules. May we rename this to something like ortmodule_module or simply ortmodule?
import onnxruntime.training.ortmodule as ortmodule

Reply:
I chose `training_ortmodule` because `ortmodule` is used a variable in one unit test. I can change.